maintainer/channels.py

```diff
@@ -69,41 +65,36 @@ def channels_to_quarantine(self):
         Method is using channels_detailed_information for getting seconds_empty
         and channel_topic values in order to specify if there should be put
         status of quarantine on channels that meet certain condition.
-        There is no input taken from outside of class.
-        Output: Method is returning short information that it found channels
-        that needed attention.
+        Method is returning log in which we can see which channel is quarantined
+        by it.
         '''
         channels_detailed_info = self.channels_detailed_information()
         for channel_id in channels_detailed_info:
             channel = channels_detailed_info[channel_id]
-            if int(channel["seconds_empty"]) > self.qtime:
+            if int(channel["seconds_empty"]) > self.qtime and channel[
+                    "channel_topic"] != 'protected':
```

Comment:
What if channel has different topic than just 'protected'? Eg 'protected by bot' or something like this?

Reply:
dont know how to do it

Reply:
So the basic problem here is that you have assumed that topic contains some predefined strings. The way the current maintainer works is that it checks if there is a desired substring inside a topic. 

So the better option IMO here is to check if topic contains 'protected' string, not that it equals 'protected'